### PR TITLE
Fix Michigan zip loading, Pentaho ETL script

### PR DESCRIPTION
**Pull request by pjsier**
# What's in this PR?

* Fixed loading from zip file for `zip_voters`
* Added a `buildMichigan.sh` script as a shortcut for loading in MI data

### CODE REVIEW

File: docker/buildMichigan.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+##################
+## Python script to clean up and enrich the Michigan Voter File
+docker-compose run etl python3 /national-voter-file/src/python/national_voter_file/transformers/csv_transformer.py -s mi -d /national-voter-file/data/Michigan/FOIA_Voters.zip -o /national-voter-file/data/Washington
```
Comment (slavingia) [inline docker/buildMichigan.sh]: there's a ref to `/national-voter-file/data/Washington` - unintentional?

File: src/python/national_voter_file/us_states/mi/transformer.py [hunk 2]
```diff
@@ -89,7 +89,10 @@ def zip_voters(self, voter_file_path):
         with open(voter_file_path, 'r') as infile:
             for row in infile:
                 # Yield column with spaces stripped
```
Comment (slavingia) [inline src/python/national_voter_file/us_states/mi/transformer.py]: this comment may need an update?

**Comment by pjsier** [comment]
Whoops, yeah that's a typo. It should point to Michigan

File: src/python/national_voter_file/us_states/mi/transformer.py (hunk 2 above)
Comment (pjsier) [inline src/python/national_voter_file/us_states/mi/transformer.py]: Makes sense, it only sort of applies now, so I can update that

**Comment by pjsier** [comment]
Agreed to the updated structure in a separate PR. I also think it would be good for that one to have an option for toggling prod.

The prod option would run the same commands, but without Docker, and then the local Docker versions would only load the sample files.

**Comment by slavingia** [comment]
yep, agree fully

**Pull request merged**